Contributing.md

```diff
@@ -37,46 +37,132 @@
-**2.**  Clone your forked copy of the project.
 ```
 git clone https://<your_user_name>@github.com/<your_user_name>/Internship-LMS-FrontEnd.git
 ```
-**3.** Navigate to the project directory. :file_folder:
+
+<br />
+
+<img src="imagesContributing/Step5.png" width="100%">
+
+**6.** Navigate to the project directory. :file_folder:
+
 ```
 cd Internship-LMS-FrontEnd
 ```
-**4.** Add a reference (remote) to the original repository.
+
+<br />
+
+<img src="imagesContributing/Step6.png" width="100%">
+
+**7.** Add a reference (remote) to the original repository.
+
 ```
 git remote add upstream https://github.com/praveenscience/Internship-LMS-FrontEnd
 ```
-**5.** Check the remotes for this repository.
+
+<br />
+
+<img src="imagesContributing/Step7.png" width="100%">
+
+**8.** Check the remotes for this repository.
+
 ```
 git remote -v
 ```
-**6.** Always take a pull from the upstream repository to your master branch to keep it at par with the main project(updated repository).
+
+<br />
+
+<img src="imagesContributing/Step8.png" width="100%">
+
+**9.** Always take a pull from the upstream repository to your master branch to keep it at par with the main project(updated repository).
+
 ```
 git pull upstream main
 ```
-**7.** Create a new branch.
+
+<br />
+
+<img src="imagesContributing/Step9.png" width="100%">
+
+**10.** Create a new branch.
+
 ```
 git checkout -b <your_branch_name>
 ```
-**8.** Perfom your desired changes to the code base.  
-**9.** Track your changes. :heavy_check_mark:
+
+<br />
+
+<img src="imagesContributing/Step10.png" width="100%">
+
+**11.** Perfom your desired changes to the code base.  
+**12.** Track your changes. :heavy_check_mark:
+
 ```
-git add . 
+git add .
 ```
-**10.** Commit your changes.
+
+<br />
+
+<img src="imagesContributing/Step12.png" width="100%">
+
+**13.** Commit your changes.
+
 ```
 git commit -m "Relevant message"
 ```
-**11.** Push the committed changes in your feature branch to your remote repo.
+
+<br />
+
+<img src="imagesContributing/Step13.png" width="100%">
+
+**14.** Push the committed changes in your feature branch to your remote repo.
+
 ```
 git push -u origin <your_branch_name>
 ```
-**12.** To create a pull request, click on `compare and pull requests`.  
-**13.** Add appropriate title and description to your pull request explaining your changes and efforts done.  
-**14.** Click on `Create Pull Request`.
+
+<br />
+
+<img src="imagesContributing/Step13.png" width="100%">
+
+**15.** To create a pull request, click on `compare and pull requests`.  
+<br />
+
+<img src="imagesContributing/Step15.png" width="100%">
+
+**16.** Add appropriate title and description to your pull request explaining your changes and efforts done.  
+<br />
+
+<img src="imagesContributing/Step16.png" width="100%">
+
+**17.** Click on `Create Pull Request`.
+<br />
+
+<img src="imagesContributing/Step17.png" width="100%">
+
+As always, if you have any questions regarding Git, please do reach out on Discord.
```

Comment:
Add our discord channel link. Mention Discord's only for GSSoC.
https://discord.com/channels/811724426965811210/815033852745220107

Reply:
If you could help with the images, it would be great. Since others will be referring to this, I don't want them to be misguided.I will try correcting things on my end.
For the HTML part, I am trying for a better approach, even discussed in discord. I tried it with markdown but couldn't get positive result.
Will try again and add discord link.

Reply:
Cool, let me send you the images. Content wise it's awesome.